src/vm/autotrace.cpp

```diff
@@ -0,0 +1,101 @@
+#include "common.h" // Required for pre-compiled header
+
+#ifdef FEATURE_AUTO_TRACE
+#ifdef FEATURE_PAL
+#include "pal.h"
+#endif // FEATURE_PAL
+
+HANDLE auto_trace_event;
+static size_t g_n_tracers = 1;
+#ifdef __apple__
```

Comment:
Is there a cleaner approach for this than `ifdef`ing?  Is there a macro I'm not familiar with in the code base that would handle this for us, much like `WCHAR`?